gen_pools_and_gauges.py

```diff
@@ -119,19 +116,22 @@ def main():
         gauge_info = BalPoolsGauges(chain)
         # pools
         # TODO: consider moving to query object??
-        result = process_query_swap_enabled_pools(query_swap_enabled_pools(chain))
-        if result:
-            pools[chain] = result
-
-        # gauges
-        if chain in NO_GAUGE_SUBGRAPH:
-            # no (gauge) subgraph exists for this chain
+        try:
+            result = process_query_swap_enabled_pools(query_swap_enabled_pools(chain))
+            if result:
+                pools[chain] = result
+        except Exception as e:
+            print(f"Error using core subgraph for {chain}, skipping: {e}")
             continue
```

Comment:
would skip the continue, and try gauges as well. a (generic) error in pools might not mean error in gauges